parquet-column/src/main/java/org/apache/parquet/column/ParquetProperties.java

```diff
@@ -266,6 +269,10 @@ public int getMaxBloomFilterBytes() {
     return maxBloomFilterBytes;
   }
 
+  public boolean getColumnIdResolution() {
```

Comment:
I prefer to use the verb from the setting, rather than adding `get` as a second verb. Code will read more easily if you use `boolean resolveColumnsByID()`. You can also add "should" or "is" to denote that the return is a boolean if you like that convention.